powerlevel9k.zsh-theme

```diff
@@ -337,6 +337,47 @@ prompt_background_jobs() {
   fi
 }
 
+# Segment that indicates usage level of current partition.
+set_default POWERLEVEL9K_HDD_USAGE_ONLY_WARNING false
+set_default POWERLEVEL9K_HDD_USAGE_WARNING_LEVEL 90
+set_default POWERLEVEL9K_HDD_USAGE_CRITICAL_LEVEL 95
+prompt_hdd_usage() {
+  local current_state="unknown"
+  typeset -AH hdd_usage_forecolors
+  hdd_usage_forecolors=(
+    'normal'        'yellow'
+    'warning'       "$DEFAULT_COLOR"
+    'critical'      'white'
+  )
+  typeset -AH hdd_usage_backcolors
+  hdd_usage_backcolors=(
+    'normal'        $DEFAULT_COLOR
+    'warning'       'yellow'
+    'critical'      'red'
+  )
+
+  local level="${$(df -k . | sed -n '2p' | awk '{ print $5 }')%%\%}"
+
+  if [ "$level" -ge "$POWERLEVEL9K_HDD_USAGE_WARNING_LEVEL" ]; then
+    current_state='warning'
+    if [ "$level" -ge "$POWERLEVEL9K_HDD_USAGE_CRITICAL_LEVEL" ]; then
+        current_state='critical'
+    fi
+  else
+    if [[ "$POWERLEVEL9K_HDD_USAGE_ONLY_WARNING" != false ]]; then
+        return
```

Comment:
In preparation for merging that in #344 , it would be better to set `level` to an empty value. It won't get printed anyway, as this is checked later on.